tests/foreman/ui/test_oscapcontent.py

```diff
@@ -0,0 +1,95 @@
+from robottelo.constants import (
+    ANY_CONTEXT,
+    OSCAP_CONTENT_FILE,
+    OSCAP_DEFAULT_CONTENT,
+)
+from robottelo.datafactory import valid_data_list
+from robottelo.helpers import get_data_file
+from robottelo.test import UITestCase
+from robottelo.ui.factory import set_context, make_oscapcontent
+from robottelo.ui.session import Session
+
+
+class OpenScapContent(UITestCase):
+    """Implements Oscap Content tests in UI."""
+
+    @classmethod
+    def setUpClass(cls):
+        super(OpenScapContent, cls).setUpClass()
+        cls.content_path = get_data_file(OSCAP_CONTENT_FILE)
+
+    def test_positive_create_oscapcontent(self):
```

Comment:
ok will update, I was thinking about the existing screenshot functionality. But I think we can work on that to update it and make use of the Class Name for it to categorize things when all the testcases from different module will have the same name like `test_positive_create`.